src/sizzle.js

```diff
@@ -505,8 +505,7 @@ setDocument = Sizzle.setDocument = function( node ) {
 
 	// Support: IE 9 - 11
```

Comment:
I added `Edge` but left the build number off because it's not resolved, so open ended.